azure-container-apps/docs/10-migrate-using-az-spring-export.md

```diff
@@ -0,0 +1,296 @@
+        --registry-server ${ACR_LOGIN_SERVER} \
+        --registry-username ${ACR_NAME} \
+        --registry-password ${ACR_PASSWORD} \
+        --ingress internal \
+        --target-port 8080
+az containerapp up \
+        --name ${PAYMENT_SERVICE} \
+        --environment ${SOURCE_ASA_NAME} \
+        --image ${ACR_LOGIN_SERVER}/${PAYMENT_SERVICE_APP}:${APP_IMAGE_TAG} \
+        --resource-group ${RESOURCE_GROUP} \
+        --subscription ${SUBSCRIPTION} \
+        --registry-server ${ACR_LOGIN_SERVER} \
+        --registry-username ${ACR_NAME} \
+        --registry-password ${ACR_PASSWORD} \
+        --ingress internal \
+        --target-port 8080
+az containerapp up \
+        --name ${ORDER_SERVICE} \
+        --environment ${SOURCE_ASA_NAME} \
+        --image ${ACR_LOGIN_SERVER}/${ORDER_SERVICE_APP}:${APP_IMAGE_TAG} \
+        --resource-group ${RESOURCE_GROUP} \
+        --subscription ${SUBSCRIPTION} \
+        --registry-server ${ACR_LOGIN_SERVER} \
+        --registry-username ${ACR_NAME} \
+        --registry-password ${ACR_PASSWORD} \
+        --ingress internal \
+        --target-port 8080
+az containerapp up \
+        --name ${CART_SERVICE} \
+        --environment ${SOURCE_ASA_NAME} \
+        --image ${ACR_LOGIN_SERVER}/${CART_SERVICE_APP}:${APP_IMAGE_TAG} \
+        --resource-group ${RESOURCE_GROUP} \
+        --subscription ${SUBSCRIPTION} \
+        --registry-server ${ACR_LOGIN_SERVER} \
+        --registry-username ${ACR_NAME} \
+        --registry-password ${ACR_PASSWORD} \
+        --ingress internal \
+        --target-port 8080
+az containerapp up \
+        --name ${FRONTEND} \
+        --environment ${SOURCE_ASA_NAME} \
+        --image ${ACR_LOGIN_SERVER}/${FRONTEND_APP}:${APP_IMAGE_TAG} \
+        --resource-group ${RESOURCE_GROUP} \
+        --subscription ${SUBSCRIPTION} \
+        --registry-server ${ACR_LOGIN_SERVER} \
+        --registry-username ${ACR_NAME} \
+        --registry-password ${ACR_PASSWORD} \
+        --ingress internal \
+        --target-port 8080
+az containerapp up \
+        --name ${IDENTITY_SERVICE} \
+        --environment ${SOURCE_ASA_NAME} \
+        --image ${ACR_LOGIN_SERVER}/${IDENTITY_SERVICE_APP}:${APP_IMAGE_TAG} \
+        --resource-group ${RESOURCE_GROUP} \
+        --subscription ${SUBSCRIPTION} \
+        --registry-server ${ACR_LOGIN_SERVER} \
+        --registry-username ${ACR_NAME} \
+        --registry-password ${ACR_PASSWORD} \
+        --ingress internal \
+        --target-port 8080
+```
+
+### 4. Correct the probe port
+Change the health probe (liveness and readiness) port of following container app from `80` to `8080` on portal page `Application` > `Containers`.
+- catalog-service
+- payment-service
+- order-service
+- cart-service
+- frontend
+- identity-service
+> Note: There is no CLI command available to enable health probe for container apps.
+
+## Verify the migrated Azure Container Apps resource
+### Get the gateway URL
```

Comment:
We need to tell customer gateway component is still preview. For the time being, we recommend user to use self-hosted version. provide link for their reference.